terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
.//tmp_files/8604.sh: line 3: 12353 Aborted                 speed3d_r2c fftw float-long 1400 1400 1400 -a2a -r2c_dir 0 -outgrid 256 4 1 -no-gpu-aware -n5
[NID 01983] 2023-07-21 07:40:18 Apid 28478044: initiated application termination
Application 28478044 exit codes: 134
Application 28478044 exit signals: Killed
Application 28478044 resources: utime ~20s, stime ~65s, Rss ~2791984, inblocks ~4062, outblocks ~0
